### ISSUE TITLE
Develop UplinkUserLinkManager to work with link.scarsz.me

### ISSUE THREAD

**Issue by VTCAKAVSMoACE** [DiscordSRV/DiscordSRV2-Core#3]
I don't really get how this works so @Scarsz you got this

**Comment by Scarsz** [comment]
Servers themselves aren’t allowed to just say who links with who on https://link.scarsz.me, to do that would be a security issue.

When people try to link their accounts and the server is in global mode, they should just be told to go to https://link.scarsz.me for instructions.

**Comment by VTCAKAVSMoACE** [comment]
Yeah I'm saying perform the fetches, we just NOOP the (un)link events
